src/python/pants/backend/python/goals/coverage_py.py

```diff
@@ -132,6 +132,9 @@ class CoverageSubsystem(PythonToolBase):
             A list of Python modules or filesystem paths to use in the coverage report, e.g.
             `['helloworld_test', 'helloworld/util/dirutil']`.
 
+            For including files without any test in coverage calculation pass paths instead of modules.
```

Comment:
This all looks good to me, save for this point. What does happen of you pass a filter of modules or a filter with mixed values - some modules and some paths? This new sentence seems fine iff the answer to these questions is nothing blows up, but coverage output may not be what you want. It might be good to add a test case for each of these though.

Reply:
Yeah makes sense. I tried to modify this test to use just module name and it actually doesn't work if we have provide a relative path also i.e. if 
```
filter = ["path/to/module1", "module2"]
```
Then we only see filtering working for `module1`.

I would have actually preferred to somehow figure out these paths for the case when use only provides a module. Is this possible to do so? My thinking is if we can figure out correct source root for a given module, this will be just concatenating those. Not sure how easy that is. If not, we change the docs to stress that you can only use such relative paths.

Reply:
This sort of calculation is exactly what dependency inference has to do. Take a list of modules and, for each one, determine if it is provided by a file in the repo. I'd look there. You should be able to reuse code but you probably can re-use whole rules by requesting the right types.